src/App.css

```diff
@@ -1,3 +1,26 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+body {
+  background-image: url(../src/Assets/books-bg.png);
```

Comment:
- [ ] This file can't be found in your app directory, this produces an error kindly fix this so your app is running without issues

![Screenshot_20221112_222642](https://user-images.githubusercontent.com/25227022/201495179-f0fe9093-2a08-4352-a0e1-a874a5fbfeb8.png)